go/billing/tests/test_api.py

```diff
@@ -50,6 +53,8 @@ def setUp(self):
 
     @inlineCallbacks
     def tearDown(self):
+        yield super(BillingApiTestCase, self).tearDown()
+
         for table in ('billing_lowcreditnotification', 'billing_transaction',
                       'billing_messagecost', 'billing_tagpool',
                       'billing_account'):
```

Comment:
I wonder whether we still need to manually clean up these tables if we're always using ``transaction=True``?